### ISSUE TITLE
Source Pipedrive: Sync Error

### ISSUE THREAD

**Issue by luketower** [airbytehq/airbyte#18785]
## Environment
- **Airbyte version**:  example is 0.40.15
- **OS Version / Instance**: AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Pipedrive 0.1.13
- **Destination Connector and version**: Snowflake 0.4.38
- **Step where error happened**: Sync job

## Current Behavior
I'm seeing the following error when syncing:
```
Database Error in model DEALS_SCD (models/generated/airbyte_incremental/scd/PIPEDRIVE_SCHEMA/DEALS_SCD.sql)
  100108 (22007): Time '2021-12-07 04:26:52' is not recognized
  compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_incremental/scd/PIPEDRIVE_SCHEMA/DEALS_SCD.sql
```
Other tables are syncing fine.  This is the only issue.

## Expected Behavior
Sync completes successfully with no errors.

## Logs
[e2e813b5_f698_4b4e_b4ae_4d3126014d9b_logs_57_txt.txt](https://github.com/airbytehq/airbyte/files/9912396/e2e813b5_f698_4b4e_b4ae_4d3126014d9b_logs_57_txt.txt)

## Steps to Reproduce
1. Setup Pipedrive source 
2. Setup Snowflake destination
3. Attempt sync

## Are you willing to submit a PR?
Unfortunately, I am unable to do so at the moment.  Any help debugging is appreciated.

**Comment by natalyjazzviolin** [comment]
@luketower please edit the initial comment to include connector versions as well! Thanks!

**Comment by luketower** [comment]
@natalyjazzviolin Sorry about that!  Updated the issue with the source and destination versions.

**Comment by jdebner** [comment]
Just waned to check if there's any update or suggestions. I've discovered that I can sometimes attempt to find the date field that has an error by checking date properties and changing them, but it eventually hits another error with a different date.

I'm happy to remove something from our pipedrive database if that'll get us unblocked, but I can't determine what this date even represents.

**Comment by grubberr** [comment]
I suspect the problem can be here
https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pipedrive/source_pipedrive/schemas/deals.json#L166

Airbyte testing data have "null" for all records for this field and I cannot reproduce this problem for now.

Keeping an eye on this problem ...

**Comment by grubberr** [comment]
Fixed in this PR https://github.com/airbytehq/airbyte/pull/23789

**Comment by jdebner** [comment]
@grubberr - FYI I bumped our version to 0.1.16 and unfortunately I'm still seeing this error.

**Comment by jdebner** [comment]
Same logs as before

```
2023-03-14 18:26:13 [42mnormalization[0m > [31mCompleted with 1 error and 0 warnings:[0m
2023-03-14 18:26:13 [42mnormalization[0m > [33mDatabase Error in model DEALS_SCD (models/generated/airbyte_incremental/scd/PIPEDRIVE_SCHEMA/DEALS_SCD.sql)[0m
2023-03-14 18:26:13 [42mnormalization[0m >   100108 (22007): Time '2022-08-03 15:55:31' is not recognized
2023-03-14 18:26:13 [42mnormalization[0m >   compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_incremental/scd/PIPEDRIVE_SCHEMA/DEALS_SCD.sql
2023-03-14 18:26:13 [42mnormalization[0m > Done. PASS=39 WARN=0 ERROR=1 SKIP=1 TOTAL=41```

**Comment by grubberr** [comment]
@jdebner usually even after schema fix you still need to press 'refresh source schema'

**Comment by jdebner** [comment]
@grubberr - Thanks for the guidance, I refreshed the source schema and that error is now resolved! 🎉 

But I'm now running into another error where multiple streams are erroring with timezone offset issues. Would you like me to create a new issue to track?

```
2023-03-15 04:38:41 [44msource[0m > Syncing stream: organization_fields 
2023-03-15 04:38:41 [1;31mERROR[m c.n.s.DateTimeValidator(tryParse):82 - Invalid date-time: No zone offset information found
2023-03-15 04:38:41 [1;31mERROR[m c.n.s.DateTimeValidator(tryParse):82 - Invalid date-time: No zone offset information found
2023-03-15 04:38:41 [1;31mERROR[m c.n.s.DateTimeValidator(tryParse):82 - Invalid date-time: No zone offset information found
2023-03-15 04:38:41 [1;31mERROR[m c.n.s.DateTimeValidator(tryParse):82 - Invalid date-time: No zone offset information found
2023-03-15 04:38:41 [1;31mERROR[m c.n.s.DateTimeValidator(tryParse):82 - Invalid date-time: No zone offset information found```

**Comment by bsilvercreek** [comment]
To add to this it looks like there is an issue that @jdebner is dealing with that is caused by a field that airbyte says isn't there but it is.

Is there a way we could call out which table is causing the issue of not being able to find update_time

<img width="1497" alt="Screenshot 2023-05-17 at 4 39 54 PM" src="https://github.com/airbytehq/airbyte/assets/132015427/c8e2f1a9-d399-493c-a967-5f4ad1e3b176">
